src/wallet/api/CMakeLists.txt

```diff
@@ -71,6 +71,8 @@ target_link_libraries(wallet_api
     mnemonics
     ${LMDB_LIBRARY}
     ${Boost_CHRONO_LIBRARY}
+    ${Boost_LOCALE_LIBRARY}
```

Comment:
These were missing from wallet_api.

https://github.com/monero-project/monero-gui/pull/3448

Once this is merged these can be removed from the GUI side.